arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -351,6 +364,11 @@ bool BenchFeature::report(ClientFeature& client, std::vector<BenchRunResult> res
     output = results[0];
   }
   printResult(output);
+  std::cout <<
+    " Min Request time: " << minTime << std::endl <<
```

Comment:
Shall we put all values into the same line? This looks a bit wasteful, spacewise.

Reply:
had that first, liked it better this way.